pkg/controller/config/v1alpha1/types.go

```diff
@@ -0,0 +1,22 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ExtensionConfig is the configuration for the os-coreos extension.
+type ExtensionConfig struct {
```

Comment:
Suggestions for a clearer config struct (as yaml example):
```yaml
ntp: # uses defaults if not set
    daemon: "ntpd" or "systemd-timesyncd"  # systemd-timesyncd is default and it does not support setting servers
    servers: ["a", "b"] # must be set when using ntpd
```

Reply:
That is even more confusing to me. Counter suggestion

```yaml
time:
   daemon: "ntpd" or "systemd-timesyncd"  # systemd-timesyncd is default and it does not support setting servers
   ntpd:
     servers: ["a", "b"] # must be set when using ntpd